docs/language-server/protocol-language-server.md

```diff
@@ -151,7 +152,6 @@ transport formats, please look [here](./protocol-architecture).
   - [`ModuleNotFoundError`](#modulenotfounderror)
   - [`VisualisationNotFoundError`](#visualisationnotfounderror)
   - [`VisualisationExpressionError`](#visualisationexpressionerror)
```

Comment:
Surely this entry should go as well. Not sure why it was duplicated.

Reply:
`VisualisationExpressionError` is an error reply when the visualization expression is compiled to an Enso function. Evaluation error is a notification sent during the program execution when the computed value is applied to the function.